compiler/lib/src/main/scala/codegen/LocateDefsFppWriter.scala

```diff
@@ -50,6 +50,16 @@ object LocateDefsFppWriter extends AstVisitor with LineUtils {
       data.members.flatMap(matchComponentMember(s1, _))
   }
 
+  override def defStateMachineAnnotatedNode(
+    s: State,
+    aNode: Ast.Annotated[AstNode[Ast.DefStateMachine]]
+  ) = {
+    val (_, node, _) = aNode
+    val data = node.data
+    val s1 = s.copy(scopeNameList = data.name :: s.scopeNameList)
```

Comment:
The line `val s1 = ...` doesn't do anything. Let's delete it. We don't need to visit the members of a state machine because (1) there aren't any in Phase 1 and (2) even in Phase 2 they won't contain any definitions that have locations.

Reply:
Fixed